examples/abitest/module/rust/src/lib.rs

```diff
@@ -573,21 +604,33 @@ impl FrontendNode {
             );
         }
         // Consume the pending message.
-        let mut buffer = Vec::<u8>::with_capacity(5);
+        let mut buffer = Vec::with_capacity(5);
+        let mut recv_len = 0u32;
         let mut handles = Vec::with_capacity(5);
-        expect_eq!(
-            OakStatus::OK,
-            oak::channel_read(in_channel, &mut buffer, &mut handles)
-        );
-        expect_eq!(3, buffer.len());
-        expect_eq!(0, handles.len());
+        let mut recv_handles = 0u32;
+        unsafe {
+            expect_eq!(
+                OakStatus::OK.value() as u32,
+                oak::wasm::channel_read(
```

Comment:
Why the change from `oak::channel_read` to `oak::wasm::channel_read` ?

Reply:
I was trying to switch to exclusively using `oak::wasm` functions in `_raw` tests and `oak::` in others. The new opaque type requires that non-raw functions use the opaque `oak::Handle` type as the handles can't be unwrapped, but the `_raw` functions can mix both I guess. It seemed therefore more consistent to only use `oak::wasm` functions in the `_raw` functions.

Reply:
OK, that makes sense.